tla-pp/src/main/scala/at/forsyte/apalache/tla/pp/passes/ConfigurationPassImpl.scala

```diff
@@ -169,6 +202,47 @@ class ConfigurationPassImpl @Inject()(val options: WriteablePassOptions,
       case None =>
         () // this is fine, invariants are optional
     }
+
+    options.get[List[String]]("checker", "temporalProps") match {
+      case Some(props) =>
+        props.foreach(assertDecl("a temporal property", _))
+
+      case None =>
+        () // this is fine, temporal properties are not supported anyway
+    }
+  }
+
+  /**
+    * Extract Init and Next from the spec definition that has the canonical form Init /\ [Next]_vars /\ ...
+    * @param module TLA+ module
+    * @param specName the name of the specification definition
+    * @return the pair (Init, Next)
+    */
+  private def extractFromSpec(module: TlaModule, contextName: String, specName: String): (String, String) = {
+    module.operDeclarations.find(_.name == specName) match {
+      case None =>
+        throw new ConfigurationError(s"$contextName: Operator $specName not found (used as SPECIFICATION)")
+
+        // the canonical form: Init /\ [Next]_vars /\ ...
+      case Some(TlaOperDecl(_, List(),
+        OperEx(TlaBoolOper.and,
+          // Init
+          OperEx(TlaOper.apply, NameEx(init)),
+          // [Next]_vars
+          OperEx(TlaActionOper.stutter,
+            // Next
+            OperEx(TlaOper.apply, NameEx(next)),
+            // vars
+            _*
+          ), ///
```

Comment:
Out of curiosity, what's the `///` meant to signal here?

Reply:
I sometimes use it to make the closing parenthesis visible

Reply:
Look visible to me :P